### PR TITLE
Add examples to roxygen docs

### PR DESCRIPTION
**Pull request by olgamie**
Short description (with a reference to an issue).
#243 

**DoD**

- [ ] Major project work has a corresponding task. If there’s no task for what you are doing, create it. Each task needs to be well defined and described.

- [ ] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.

- [ ] No new error or warning messages are introduced.

- [ ] All interaction with a semantic functions, examples and docs are written from the perspective of the person using or receiving it. They are understandable and helpful to this person.

- [ ] If the change affects code or repo sctructure, README, documentation and code comments should be updated. 

- [ ] All code has been peer-reviewed before merging into any main branch.

- [ ] All changes have been merged into the main branch we use for development (develop).

- [ ] Continuous integration checks (linter, unit tests) are configured and passed.

- [ ] Unit tests added for all new or changed logic.

- [ ] All task requirements satisfied. The reviewer is responsible to verify each aspect of the task.

- [ ] Any added or touched code follows our style-guide.

### CODE REVIEW

File: R/rating.R [hunk 1]
```diff
@@ -58,6 +60,29 @@ rating_input <- function(input_id, label = "", value = 0, max = 3, icon = "star"
...
+#' ## Only run examples in interactive R sessions
+#' if (interactive()) {
+#' library(shiny)
+#' library(shiny.semantic)
+#'
+#' ui <- shinyUI(
+#'   semanticPage(
+#'     rating_input("rate", "How do you like it?", max = 5,
+#'                  icon = "heart", color = "yellow"),
+#'     numeric_input("numeric_in","", 0, min = 0, max = 5)
```
Comment (dokato) [inline R/rating.R:73]: please add space after coma `numeric_input("numeric_in", "", 0, min = 0, max = 5)`

File: R/rating.R [hunk 2]
```diff
@@ -58,6 +60,29 @@ rating_input <- function(input_id, label = "", value = 0, max = 3, icon = "star"
...
+#'   )
+#' )
+#' server <- function(session, input, output) {
+#'   observeEvent(input$numeric_in,{
```
Comment (dokato) [inline R/rating.R:77]: space `input$numeric_in, {`

File: R/rating.R [hunk 3]
```diff
@@ -58,6 +60,29 @@ rating_input <- function(input_id, label = "", value = 0, max = 3, icon = "star"
...
+#'
```
Comment (dokato) [inline R/rating.R:78]: could you remove empty lines?

File: R/semanticPage.R [hunk 4]
```diff
@@ -121,6 +121,28 @@ check_semantic_theme <- function(theme_css, full_url = TRUE) {
...
+#' @examples
+#' ## Only run examples in interactive R sessions
+#' if (interactive()) {
+#' library(shiny)
+#' library(shiny.semantic)
+#'
+#' ui <- semanticPage(
+#'   title = "Hello Shiny Semantic!",
+#'   label("Number of observations:",
+#'         slider_input("obs", value = 500, min = 0, max = 1000)),
```
Comment (dokato) [inline R/semanticPage.R:133]: `slider_input` shouldn't be wrapped by label tag:
```r
  label("Number of observations:"),
  slider_input("obs", value = 500, min = 0, max = 1000),
```

File: R/semanticPage.R [hunk 5]
```diff
@@ -121,6 +121,28 @@ check_semantic_theme <- function(theme_css, full_url = TRUE) {
...
+#'   segment(
+#'     plotOutput("dist_plot"))
```
Comment (dokato) [inline R/semanticPage.R:135]: could you fix bracketing:
```r
  segment(
    plotOutput("dist_plot")
  )
```

File: R/slider.R [hunk 6]
```diff
@@ -114,6 +114,25 @@ rangeInput <- function(inputId, label, min, max, value, step = 1, width = NULL,
...
+#'     ui = semanticPage(
+#'       p("The first slider controls the second"),
+#'       slider_input("control", "Controller:", min = 0, max = 20, value = 10,
+#'                    step = 1),
+#'       slider_input("receive", "Receiver:", min = 0, max = 20, value = 10,
+#'                    step = 1)
+#'     ),
+#'     server = function(input, output, session) {
+#'       observe({
+#'         val <- input$control
```
Comment (dokato) [inline R/slider.R:130]: is `val` needed here, eg. : `update_slider(session, "receive", value = input$control)`

Comment (olgamie) [reply R/slider.R]: fixed

File: R/semanticPage.R (hunk 5 above)
Comment (olgamie) [reply R/semanticPage.R]: fixed

File: R/semanticPage.R (hunk 4 above)
Comment (olgamie) [reply R/semanticPage.R]: fixed

File: R/rating.R (hunk 3 above)
Comment (olgamie) [reply R/rating.R]: fixed

File: R/rating.R (hunk 2 above)
Comment (olgamie) [reply R/rating.R]: fixed

File: R/rating.R (hunk 1 above)
Comment (olgamie) [reply R/rating.R]: fixed

File: R/menu.R [hunk 7]
```diff
@@ -26,7 +26,8 @@ digits2words <- function(number) {
 #' it gets highlighted), default empty (\code{""})
 #' @param icon non-mandatory parameter with icon name
 #'
-#' @return shiny.tag.a link
+#' @return shiny tag a link
```
Comment (dokato) [inline R/menu.R:29]: no 'a'

File: R/semanticPage.R [hunk 8]
```diff
@@ -121,6 +121,29 @@ check_semantic_theme <- function(theme_css, full_url = TRUE) {
...
+#' library(shiny)
+#' library(shiny.semantic)
+#'
+#' ui <- semanticPage(
+#'   title = "Hello Shiny Semantic!",
+#'   label("Number of observations:"),
+#'   slider_input("obs", value = 500, min = 0, max = 1000),
+#'   segment(
+#'     plotOutput("dist_plot")
+#'     )
```
Comment (dokato) [inline R/semanticPage.R:136]: fix indentation such that it is at the segment level.

**Pull request merged**